packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -117,6 +117,8 @@ export const wrapRootElement = ({ element }, pluginOptions) => {
           aiAssistantApiBaseUrl: pluginOptions.aiAssistantApiBaseUrl,
           aiAssistantTopbarButton:
             pluginOptions.aiAssistantTopbarButton || false,
+          enableCanonicalUrls:
```

Comment:
Why not just...
```suggestion
          enableCanonicalUrls: pluginOptions.enableCanonicalUrls,
```
?

Reply:
That's because pluginOptions.enableCanonicalUrls can be `undefined` but we want to default to `true` instead of `false`. That's the reason of the weird syntax